### DISCUSSION TITLE
Some minor mistakes in the user guide.

### DISCUSSION THREAD

**Discussion by Jackchou00** [colour-science/colour#1313] [General]
The user guide is really helpful when I try to comprehensively understand the library at the beginning. However, I have found some issues in the documentation, so if its outdated or have some minor mistakes, I listed two examples below.

User guide URL: https://colour.readthedocs.io/en/master/index.html

## About 3.20.3 Blackbody

``` python
blackbody_sds = [
    colour.sd_blackbody(i, colour.SpectralShape(0, 10000, 10))
    for i in range(1000, 15000, 1000)
]
plot_multi_sds(
    blackbody_sds,
    y_label="W / (sr m$^2$) / m",
    plot_kwargs={"use_sd_colours": True, "normalise_sd_colours": True},
    legend_location="upper right",
    bounding_box=(0, 1250, 0, 2.5e6),
)
```

Output:
```
AssertionError: Wavelengths must be positive real numbers!
```

Meanwhile, if using(10, 10000, 10) instead of (0, 10000, 10), the code will work fine but can't get the same result as the documentation (All lines are red).

## About 3.20.5 Luminous Efficiency Functions

``` python
sd_mesopic_luminous_efficiency_function = (
    colour.sd_mesopic_luminous_efficiency_function(0.2)
)
plot_multi_sds(
    (
        sd_mesopic_luminous_efficiency_function,
        colour.PHOTOPIC_LEFS["CIE 1924 Photopic Standard Observer"],
        colour.SCOTOPIC_LEFS["CIE 1951 Scotopic Standard Observer"],
    ),
    y_label="Luminous Efficiency",
    legend_location="upper right",
    y_tighten=True,
    margins=(0, 0, 0, 0.1),
)
```

Output:
```
AttributeError: module 'colour' has no attribute 'PHOTOPIC_LEFS'
```

Because there is no `PHOTOPIC_LEFS` and `SCOTOPIC_LEFS` in the `colour` module. Use the following code instead:

``` python
colour.SDS_LEFS["CIE 1924 Photopic Standard Observer"]
colour.SDS_LEFS["CIE 1951 Scotopic Standard Observer"]
```

The code above will work fine and give the same result as the documentation.

**Comment by thomasmansencal** [comment]
Hello @Jackchou00,

Thanks for the issue, appreciated! We unfortunately do not have a way to test all those examples so sometimes they do break!

I will fix them.

Cheers,

Thomas

**Reply by thomasmansencal** [reply]
I did a pass over most of the examples and fixed / updated a few of them, thanks again!

**Reply by Jackchou00** [reply]
@KelSolaar

Thanks for the quick response and for addressing the issues! I've reviewed the latest commit and the bugs are indeed fixed. Your efforts are greatly appreciated.

Looking forward to contributing more in the future.

Best regards,
Jackchou00
